pyvista/core/pointset.py

```diff
@@ -1457,7 +1457,7 @@ def cell_connectivity(self) -> np.ndarray:
         carr = self.GetCells()
         if _vtk.VTK9:
             return _vtk.vtk_to_numpy(carr.GetConnectivityArray())
-        raise AttributeError('Install vtk>=9.0.0 for `cell_connectivity`\n'
+        raise VTKVersionError('Install vtk>=9.0.0 for `cell_connectivity`\n'
                              'Otherwise, use the legacy `cells` method')
```

Comment:
This would need to be indented with a space but we'll have `black` running soon which will rewrite this line anyway.

Reply:
The `black` PR is in and I've merged `main` with your branch to resolve corresponding merge conflicts.